commit e8ae212aca78e332a03e79a4dacc75cfffa43011
Author: Breno Leitao <breno.leitao@gmail.com>
Date:   Tue April 04 14:12:11 2017 -0300

    Include limits.h if you want to use PATH_MAX

    timew FTBFS on ppc64le due to lack of PATH_MAX being implicitly defined.
    On ppc64le, if you wish to have access to this macro, you must have
    limits.h included explicitly.

    Signed-off-by: Breno Leitao <breno.leitao@gmail.com>

Index: timew/src/timew-1.0.0/src/libshared/src/FS.cpp
===================================================================
--- a/src/libshared/src/FS.cpp
+++ b/src/libshared/src/FS.cpp
@@ -41,7 +41,7 @@
 #include <shared.h>
 #include <format.h>
 
-#if defined SOLARIS || defined NETBSD || defined FREEBSD
+#if defined SOLARIS || defined NETBSD || defined FREEBSD || !defined(PATH_MAX)
 #include <limits.h>
 #endif
 
